### PR TITLE
Add instructions for Traefik

### PR DESCRIPTION
**Pull request by grampelberg**

### CODE REVIEW

File: linkerd.io/resources/_gen/assets/scss/scss/index.scss_5ad6f408b0e3e473c748aac88af0ea18.content [hunk 1]
```diff
@@ -0,0 +1,6508 @@
+/*!
```
Comment (klingerf) [inline linkerd.io/resources/_gen/assets/scss/scss/index.scss_5ad6f408b0e3e473c748aac88af0ea18.content]: Hmm, should this file be in this diff? Seems like maybe it was checked in by accident?

Comment (grampelberg) [reply linkerd.io/resources/_gen/assets/scss/scss/index.scss_5ad6f408b0e3e473c748aac88af0ea18.content]: *grumble* no. Netlify is using an older version of hugo that can't do this generation. So, for the time being we need to check these generated files in until they fix it on their end.

Comment (klingerf) [reply linkerd.io/resources/_gen/assets/scss/scss/index.scss_5ad6f408b0e3e473c748aac88af0ea18.content]: Ahhh, ok, hmm. Does every update to the docs require a change to the generated files? Or rather, what is it about the docs change you're making in this branch that produces a new generated file? Is it a result of the shortcode that you're adding?

Comment (grampelberg) [reply linkerd.io/resources/_gen/assets/scss/scss/index.scss_5ad6f408b0e3e473c748aac88af0ea18.content]: That's a good question, it doesn't show up in the file anywhere...

Comment (grampelberg) [reply linkerd.io/resources/_gen/assets/scss/scss/index.scss_5ad6f408b0e3e473c748aac88af0ea18.content]: Looks like it does only change on css changes ... there's just a funky state right now that I need to figure out. Updated shrink the changes based off a rebase on master.

**Pull request merged**